pkg/network/server.go

```diff
@@ -285,7 +285,7 @@ func (s *Server) Start() {
 	setServerAndNodeVersions(s.UserAgent, strconv.FormatUint(uint64(s.id), 10))
 	setNeoGoVersion(config.Version)
 	setSeverID(strconv.FormatUint(uint64(s.id), 10))
-	s.run()
+	go s.run()
```

Comment:
Adjust documentation to `Start`. Add a note about the fact that the caller should wait for `Start` to finish for normal server operation.